source/flow/tcp/tcp_flow.hpp

```diff
@@ -48,7 +48,7 @@ class TcpFlow : public IFlow,
             m_flag_manager.get_flag(packet, packet_type_label) ==
                 PacketType::ACK) {
             // ACK delivered to source device; calculate metrics, update
-            // internal state
+            // internal state(ь_ - packet.delivered_at_origin) / rtt
```

Comment:
If I understand this correctly, we dont need this one here
```suggestion
            // internal state
```